target/linux/ramips/dts/mt7620a_netis_wf2770.dts

```diff
@@ -0,0 +1,161 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "netis,wf2770", "ralink,mt7620a-soc";
+	model = "NETIS WF2770";
+
+	leds {
+		compatible = "gpio-leds";
+
+		wlan {
```

Comment:
Only one led for both radios? What is stock behavior?

Reply:
It turns on if at least one of the wlans are on. I don't know how to "OR" two netdev so I just used phy0 here.

Reply:
@CHKDSK88 Thanks for spotting this.
AFAIK two wifi triggers can't be done in OpenWrt.
I typically requested to remove the trigger in these cases, so the user can decide what he/she wants. However, I had a discussion about this without another committer recently, and he thought that it might be more helpful to have the 2.4g linked, as this would create more useful information for the user without any specific setup.
So, effectively, I'd leave this to @mans0n , as long as it's really a designated WiFi button.

Reply:
Maybe script, which changes led 2g<->5g?